src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js

```diff
@@ -578,45 +594,93 @@ export default function ApplicationForm({
 
     if (!birthLoc || !validateBirthPlace(birthLoc)) {
       setBirthPlaceError("City and country of birth are required");
+      if (!hasScrolled) {
+        scrollToRef(birthLocRef);
+        hasScrolled = true;
+      }
     }
 
     if (!phoneNum) {
       setPhoneNumberError("Primary phone number is required");
+      if (!hasScrolled) {
+        scrollToRef(phoneNumRef);
+        hasScrolled = true;
+      }
     } else if (!validatePhoneNumber(phoneNum)) {
       setPhoneNumberError("Phone number must be in the form XXX XXX-XXXX");
+      if (!hasScrolled) {
+        scrollToRef(phoneNumRef);
+        hasScrolled = true;
+      }
     }
 
     if (!email) {
       setEmailAddressError("Personal email address is required");
+      if (!hasScrolled) {
+        scrollToRef(emailRef);
+        hasScrolled = true;
+      }
     } else if (!validateEmail(email)) {
       setEmailAddressError("Email address must be in the form name@company.ca");
+      if (!hasScrolled) {
+        scrollToRef(emailRef);
+        hasScrolled = true;
+      }
     }
 
     if (!job) {
       setJobTitleError("Position/job title is required");
+      if (!hasScrolled) {
+        scrollToRef(jobRef);
+        hasScrolled = true;
+      }
     }
 
     if (defaultScheduleTypeCd === "WBSD" && !organizationLocation) {
       setOrganizationFacilityError("Organization facility is required");
+      if (!hasScrolled) {
+        scrollToRef(organizationFacilityRef);
+        hasScrolled = true;
+      }
     }
 
     if (!sameAddress && !mailingAddressLine1) {
       setMailingAddressLine1Error("Street or PO box is required");
+      if (!hasScrolled) {
+        scrollToRef(mailingAddressLine1Ref);
+        hasScrolled = true;
+      }
     }
 
     if (!sameAddress && !mailingCity) {
       setMailingCityError("City is required");
+      if (!hasScrolled) {
+        scrollToRef(mailingCityRef);
+        hasScrolled = true;
```

Comment:
what do you think of moving these two lines into a function since it is duplicated a lot throughout this file
a function like `scrollToError(errorRef)` and then call it with `scrollToError(mailingCityRef)`

Reply:
Sounds good, I'll do this.